test/test_spectral_ops.py

```diff
@@ -21,6 +21,64 @@
 fft_fn = torch.fft
 import torch.fft
 
+
+def _complex_stft(x, *args, **kwargs):
+    # Transform real and imaginary components separably
+    stft_real = torch.stft(x.real, *args, **kwargs, return_complex=True, onesided=False)
+    stft_imag = torch.stft(x.imag, *args, **kwargs, return_complex=True, onesided=False)
+    return stft_real + 1j * stft_imag
```

Comment:
torch.complex(stft_real, stft_imag) should do this, right?

Reply:
Perhaps the variable names are confusing. This is
```python
stft(x.real) + 1j * stft(x.imag)
```
where both `stft` give complex tensors.

Reply:
Aha, thank you. You're totally correct.